### PR TITLE
add missing bandcamp embeds

### PR DESCRIPTION
**Pull request by Niklink**
adds embeds for squiddles, the wanderers, and sburban jungle (brief mix)

### CODE REVIEW

**Review by GiovanH — APPROVED** [review]

**Comment by GiovanH** [comment]
Does hsmusic have this same info?

**Comment by Niklink** [comment]
This only works for embeds, hsmusic uses direct links to track pages. So no, all of the embed info in the collection is not on hsmusic.

Notes:
1- Doesn't work for tracks that were removed from albums (Bolin, SolusLunes, etc)
2- Doesn't work on bonus tracks (we don't have track IDs for those, maybe they exist but they're not public)

So this is all the missing music that can be given an embed.

File: archive/data/music.json [hunk 1]
```diff
@@ -20271,9 +20271,8 @@
     "referencedBy": [],
     "date": "2012-04-02T01:00:00.000Z",
     "directory": "temporal-shenanigans",
-    "bandcampId": 1885000843,
+    "bandcampId": null,
```
Comment (GiovanH) [inline archive/data/music.json:20274]: okay well now you're just doing the opposite of the thing

Comment (Niklink) [reply archive/data/music.json:20274]: destiny's path twists and turns, we can but walk down it

**Pull request merged**